scripts/version

```diff
@@ -6,7 +6,7 @@ fi
 
 COMMIT=$(git rev-parse --short HEAD)
 GIT_TAG=${DRONE_TAG:-$(git tag -l --contains HEAD | head -n 1)}
-
+CROSS=true
```

Comment:
Is this value used elsewhere?

Reply:
That was already configured in the build script to enable cross compilation. I just used the existing hook to make it trigger.